lib/membrane_live_web/router.ex

```diff
@@ -26,25 +30,37 @@ defmodule MembraneLiveWeb.Router do
 
   scope "/", MembraneLiveWeb do
     pipe_through(:browser)
-    pipe_through(:auth)
+    pipe_through(:auth_restricted)
     get("/me", UserInfoController, :index)
   end
 
   scope "/resources/webinars", MembraneLiveWeb do
     pipe_through(:browser)
 
-    get("/", WebinarController, :index)
     get("/:uuid", WebinarController, :show)
     get("/:uuid/products", WebinarProductController, :index)
   end
 
+  scope "/resources/webinars", MembraneLiveWeb do
+    pipe_through(:browser)
+    pipe_through(:auth_unrestricted)
+
+    get("/", WebinarController, :index)
+  end
+
   scope "/resources/recordings", MembraneLiveWeb do
     pipe_through(:browser)
 
-    get("/", RecordingsController, :index)
     get("/:uuid", RecordingsController, :show)
   end
 
+  scope "/resources/recordings", MembraneLiveWeb do
+    pipe_through(:browser)
+    pipe_through(:auth_unrestricted)
+
+    get("/", RecordingsController, :index)
```

Comment:
So now we can list all recordings, but we can watch recording only if we are authenticated?

Reply:
You can watch whatever you want but you have to have a link.
`:auth_unrestricted` means that it will let through all users (event not authenticated).